cmd/cloud-api-adaptor/cloudmgr/manager.go

```diff
@@ -1,3 +1,5 @@
+//go:build libvirt || aws || azure || ibmcloud || vsphere
```

Comment:
Why do we need this build constraint?

When someone forks this repo and wants to implement a new cloud provider, this line is also needed to be updated. If this line does not exist, I think adding a new cloud provider becomes simpler.

Reply:
not having this allows you to compile cloudmgr without setting a cloud-provider tag, which worth nothing, IIUC it is expected to be removed soon once we will switch to single binary and compile all providers
However, I don't mind removing it already